utils/spacewalk-common-channels.ini

```diff
@@ -1593,6 +1593,15 @@ checksum = sha256
 base_channels = ubuntu-16.04-pool-amd64
 repo_url = http://archive.ubuntu.com/ubuntu/dists/xenial-updates/universe/binary-amd64/
 
+[ubuntu-1604-amd64-universe-security]
```

Comment:
One question about this, for 16 and 18, we don't mention Universe at the doc **at all**.

Since it's not needed to bootstrap, it's not mentioned at the list of repos to be added via `spacewalk-common-channels but we never mention later that they can be added via  `spacewalk-common-channels` if needed.

Do we need any mention about it? I'd say yes, but that new mention should be added when the mention for Ubuntu 20.04 is added.

Reply:
I would see this as a doc feature request. It is not a required info, but nice to have.

Reply:
@mcalmer did you already create an issue for the doc team about the changes for Ubuntu20.04 and SUSE Manager? If so, give me the URL and I will add this request there.